File: lib/models/message.dart
class Message {
  final String message;
  final String name;
  final DateTime timestamp;

  Message({required this.message, required this.name, required this.timestamp});

  factory Message.fromJson(Map<String, dynamic> json) {
    return Message(
      message: json['message'],
      name: json['name'],
      timestamp: DateTime.parse(json['timestamp']),
    );
  }
}

File: server/models/message.js
const mongoose=require('mongoose');

const messageSchema= mongoose.Schema({
    name:{
        type:String,
        required:true
    },
    message:{
        type:String,
        required:true,
    },
    
    timestamp:{
        default:Date.now,
        type:Date

    },
   
});

const Message= mongoose.model('Message',messageSchema);
module.exports=Message;